### ISSUE TITLE
Usability: Unable to find installation instructions

### ISSUE THREAD

**Issue by RedSwift** [pact-foundation/docs.pact.io#7]
I tried navigating around but I couldn't seem to find any references on how to install pact on the website.

Perhaps it was because we would want pact to be language agnostic and hence didn't provide installation instructions (since each environment is different).

However, I believe it might be beneficial to add a section in the guides or docs to let users know to refer to their respective language repositories for further instructions on installation (like NPM for node users) or maybe just a summary of initial steps?

Thanks for the great work!

**Comment by bethesque** [comment]
Yes, you raise a very good point, and it's something we're aware of. At the moment, the docs just point you out to the site for each of the relevant implementations because we haven't had a way to bring all those docs back into one place. We've just migrated to a docs solution that gives us much more control over the content, so it's on my TODO list to automatically pull in the README document for each of the languages and embed them inside the docs site.

**Comment by RedSwift** [comment]
Really exciting news to hear! It's the first time i am actually posting an issue so not really sure if I am supposed to close this or leave it open.

Let me know if I need to close the issue seeing that the team will be addressing it soon :)

**Comment by bethesque** [comment]
Leave it open. It's nice having something to check off when things get done.

**Comment by bethesque** [comment]
Done Pact-JS.

https://docs.pact.io/implementation_guides/javascript/readme#installation

Now to do all the other languages...

**Comment by YOU54F** [comment]
Might be nice to add a installation section in the list

![Screenshot 2022-03-17 at 19 21 03](https://user-images.githubusercontent.com/19932401/158879616-2aef7c43-82a6-457c-896b-1177305740cd.png)

Current state of play

- [X] [Pact JS](https://docs.pact.io/implementation_guides/javascript/readme#installation)
- [ ] Pact JVM
- [X] [Pact .NET](https://docs.pact.io/implementation_guides/net/readme#installation)
- [X] [Pact Go](https://docs.pact.io/implementation_guides/go/readme#installation)
- [ ] [Pact Python](https://docs.pact.io/implementation_guides/python/readme#installation)
- [ ] [Pact Swift](https://github.com/DiUS/pact-consumer-swift/blob/master/README.md#installation)
  - Note, hosted on DiUS GH org so readme not pulled into docs see [overview](https://docs.pact.io/implementation_guides/swift)
- [ ] Pact Scala
  - Note, hosted on ITV GH org so readme not pulled into docs see [overview](https://docs.pact.io/implementation_guides/scala)
- [ ] [Pact PHP](https://github.com/pact-foundation/pact-php/#installation)
  - note readme not being pulled into docs
- [X] [Pact Ruby](https://docs.pact.io/implementation_guides/ruby/readme#installation)
- [ ] [Pact Rust](https://docs.pact.io/implementation_guides/rust#installing--getting-started)
- [ ] Pact C++
